sros2/xml_cache/xhtml-cache.xml

```diff
@@ -0,0 +1,5 @@
+<catalog xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
+  <rewriteURI
+      uriStartString="http://www.w3.org/2001/03/"
```

Comment:
Are there any other w3 documents that we use that would also have there uri prefix rewritten? Was the XML.xsd the only source of outbound network traffic?

Reply:
The manual test I ran for this was:

1) Build and test 'master' and observe that the tests pass
2) Rip the ethernet cable out of the back of my computer (or if you want to be less dramatic `sudo ifconfig eno2 down`) and observe that the tests fail
3) Build and test this branch and observe that the tests pass again
4) Plug the ethernet cable back into my computer and observe that the tests still pass

So I'm pretty confident that this was the only external resource, unless there are other schemas that aren't tested as part of the unit test

If another external resource with the above prefix is used, but a local copy is not added to the XML cache, then the lookup will fail.

I suppose it's possible for some other python program to import sros2 (thus getting its XML catalog injected into its environment), then attempt a lookup of another document with a prefix `http://www.w3.org/2001/03/something_else`  In that case, the lookup will fail.

I will change the prefix to

```
<rewriteURI
    uriStartString="http://www.w3.org/2001/03/xml.xsd"
    rewritePrefix="./xml.xsd" />
```

So that ONLY this file is cataloged.  That should prevent us from breaking other XML lookups in the same python process as sros2